login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
@alexeykazakov yeah! ignore it (late night coding). If we move it to the token pkg, we will have import cycles.